local-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+    mongodb:
+        image: autopilotpattern/mongodb:latest
+        restart: always
+        mem_limit: 512m
+        build: .
+        env_file: _env
```

Comment:
So far, nothing but log level. :disappointed:  But would be Manta info once we have backups going.